### PR TITLE
SKUs: log the return value of top level functions called from C++

### PR DESCRIPTION
**Pull request by evq**
per https://docs.rs/tracing/latest/tracing/attr.instrument.html, this will log the return value of these functions (INFO aka VLOG(2) on success, WARN aka VLOG(1) on failure.

## Submitter Checklist:

- [ ] I confirm that no [security/privacy review is needed](https://github.com/brave/brave-browser/wiki/Security-reviews) and no other type of reviews are needed, or that I have [requested](https://github.com/brave/reviews/issues/new/choose) them
- [ ] There is a [ticket](https://github.com/brave/brave-browser/issues) for my issue
- [ ] Used Github [auto-closing keywords](https://docs.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue) in the PR description above
- [ ] Wrote a good [PR/commit description](https://google.github.io/eng-practices/review/developer/cl-descriptions.html)
- [ ] Squashed any review feedback or "fixup" commits before merge, so that history is a record of what happened in the repo, not your PR
- [ ] Added appropriate labels (`QA/Yes` or `QA/No`; `release-notes/include` or `release-notes/exclude`; `OS/...`) to the associated issue
- [ ] Checked the PR locally:
  * `npm run test -- brave_browser_tests`, `npm run test -- brave_unit_tests` [wiki](https://github.com/brave/brave-browser/wiki/Tests)
  * `npm run lint`, `npm run presubmit` [wiki](https://github.com/brave/brave-browser/wiki/Presubmit-checks), `npm run gn_check`, `npm run tslint`
- [ ] Ran `git rebase master` (if needed)

## Reviewer Checklist:

- [ ] A security review [is not needed](https://github.com/brave/brave-browser/wiki/Security-reviews), or a link to one is included in the PR description
- [ ] New files have MPL-2.0 license header
- [ ] Adequate test coverage exists to prevent regressions
- [ ] Major classes, functions and non-trivial code blocks are well-commented
- [ ] Changes in component dependencies are properly reflected in `gn`
- [ ] Code follows the [style guide](https://chromium.googlesource.com/chromium/src/+/HEAD/styleguide/c++/c++.md)
- [ ] Test plan is specified in PR before merging

## After-merge Checklist:

- [ ] The associated issue milestone is set to the smallest version that the
  changes has landed on
- [ ] All relevant documentation has been updated, for instance:
  - [ ] https://github.com/brave/brave-browser/wiki/Deviations-from-Chromium-(features-we-disable-or-remove)
  - [ ] https://github.com/brave/brave-browser/wiki/Proxy-redirected-URLs
  - [ ] https://github.com/brave/brave-browser/wiki/Fingerprinting-Protections
  - [ ] https://github.com/brave/brave-browser/wiki/Brave%E2%80%99s-Use-of-Referral-Codes
  - [ ] https://github.com/brave/brave-browser/wiki/Web-Compatibility-Exceptions-in-Brave
  - [ ] https://github.com/brave/brave-browser/wiki/QA-Guide
  - [ ] https://github.com/brave/brave-browser/wiki/P3A

### CODE REVIEW

**Comment by evq** [comment]
hm looks like this failed as we're using tracing `0.1.37`, `level = `, we'll need to upgrade to `0.1.40` to use it ( though apparently added in `0.1.38`, it and `0.1.39` were yanked. )

**Comment by rillian** [comment]
> though apparently added in `0.1.38`, it and `0.1.39` were yanked.

Probably for [use-after-free in Instrument::into_inner](https://github.com/tokio-rs/tracing/pull/2765) (which we don't call).

File: third_party/rust/tracing/v0_1/crate/Cargo.toml [hunk 1]
```diff
@@ -98,16 +95,20 @@ optional = true
 version = "0.2.9"
 
 [dependencies.tracing-attributes]
-version = "0.1.23"
+version = "0.1.27"
```
Comment (rillian) [inline third_party/rust/tracing/v0_1/crate/Cargo.toml:98]: Looks like you might need [tracing-attributes 0.1.24](https://github.com/tokio-rs/tracing/blob/tracing-attributes-0.1.27/tracing-attributes/CHANGELOG.md#0124-april-24th-2023) or later as well.

Comment (rillian) [reply third_party/rust/tracing/v0_1/crate/Cargo.toml:98]: Oh, you did that already. Nevermind.

File: third_party/rust/tracing/v0_1/README.chromium [hunk 2]
```diff
@@ -2,7 +2,9 @@ Name: tracing
 URL: https://crates.io/crates/tracing
 Description: Application-level tracing for Rust.
 
-Version: 0.1.37
-Security Critical: yes
+Version: 0.1.40
+Security Critical: no
+Shipped: no
```
Comment (rillian) [inline third_party/rust/tracing/v0_1/README.chromium:7]: `Shipped: yes` is correct for this, I think?

**Review by rillian — APPROVED** [review]
lgtm

**Review by bridiver — APPROVED** [review]

**Comment by evq** [comment]
going foward with merge as CI failure is due to unrelated npm audit

**Pull request merged**